build-tools/packages/build-cli/src/library/commands/generateSourceEntrypoints.ts

```diff
@@ -0,0 +1,194 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fs from "node:fs/promises";
+import JSON5 from "json5";
+
+import type { PackageJson } from "@fluidframework/build-tools";
+import type { ExportSpecifierStructure, Node } from "ts-morph";
+import { ModuleKind, Project, ScriptKind } from "ts-morph";
+import type { TsConfigJson } from "type-fest";
+
+import type { CommandLogger } from "../../logging.js";
+
+import { ApiLevel } from "../apiLevel.js";
+import { ApiTag } from "../apiTag.js";
+import type { ExportData } from "../packageExports.js";
+import { getApiExports, getPackageDocumentationText } from "../typescriptApi.js";
+
+export const optionDefaults = {
```

Comment:
There is only one use of this object. It is only used in generate/sourceEntrypoints.ts and not even used locally. Looks like artifact of moving some code to commands instead of library.